Makefile

```
@@ -22,6 +22,13 @@ ifndef BUILD_NUMBER
 	endif
 endif
 
+ifndef DOCKER_IMAGE_REPO
+	DOCKER_IMAGE_REPO = nebulaoss/nebula
+endif
+ifndef DOCKER_IMAGE_TAG
+	DOCKER_IMAGE_TAG = latest
+endif
+
```

```suggestion
DOCKER_IMAGE_REPO ?= nebulaoss/nebula
DOCKER_IMAGE_TAG ?= latest
```

if we define it this way, you should still be able to override it.